opencti-platform/opencti-graphql/src/database/middleware.js

```diff
@@ -498,7 +499,9 @@ const convertAggregateDistributions = async (context, user, limit, orderingFunct
 export const timeSeriesEntities = async (context, user, types, args) => {
   const timeSeriesArgs = buildEntityFilters({ types, ...args });
   const { startDate, endDate, interval } = args;
-  const histogramData = await elHistogramCount(context, user, args.onlyInferred ? READ_DATA_INDICES_INFERRED : READ_DATA_INDICES, timeSeriesArgs);
+  const dataIndices = args.onlyInferred ? READ_DATA_INDICES_INFERRED : READ_DATA_INDICES;
+  const indexName = types.includes(ENTITY_TYPE_HISTORY) ? INDEX_HISTORY : dataIndices;
```

Comment:
Shouldn't the check be that the types only contains History ? I'm not sur to understand the choice that has been made of getting only History (after reading the brainstorm). 

Additionnaly, why INDEX_HISTORY and not READ_INDEX_HISTORY ?

Reply:
right for READ_INDEX_HISTORY instead of INDEX_HISTORY

not sure to understand the rest, we can have a quick talk